build.zig

```diff
@@ -211,10 +211,31 @@ fn ensureDependencySubmodule(allocator: std.mem.Allocator, path: []const u8) !vo
     if (std.process.getEnvVarOwned(allocator, "NO_ENSURE_SUBMODULES")) |no_ensure_submodules| {
         if (std.mem.eql(u8, no_ensure_submodules, "true")) return;
     } else |_| {}
+    ensureGit(allocator);
```

Comment:
I think this should be rather put here: https://github.com/hexops/mach/blob/0dabe0d49fa621ed43dc2ba092f56e54d4b117b3/build.zig#L26
There's no need to call this function everytime you invoke git